### PR TITLE
[IT] Italian Support

### LINKED ISSUE #62: Italian sentences that should be supported
If you don't feel comfortable contributing to the YAML in this repository, you can post your **Italian** (it) sentences that match [our supported intents](https://github.com/home-assistant/intents/blob/main/intents.yaml) below.

### PR DESCRIPTION
**Pull request by auanasgheps**
Hi there,
Here's my work for the Italian translation.

Fixes #62 

I'd like to be the leader of this language. I'm a daily user of Home Assistant and have translated dozens of projects.

### CODE REVIEW

**Comment by xraver** [comment]
Sorry I didn't see your work... 
we can collaborate... i've prepared a pull request... we can merge them

**Comment by auanasgheps** [comment]
@xraver I opened the PR first, so what we do?

**Comment by xraver** [comment]
You have the Priority
I can cancel mine and work with you merging mine with yours. Does it work
for you?

Il Mar 27 Dic 2022, 19:03 Oliver Cervera ***@***.***> ha
scritto:

**Comment by auanasgheps** [comment]
Sure, great!

Hopefully you added more/different sentences than me, to make the detection much better.

I was also missing some files in the test folder, I won't do the job twice and will wait for your PR. You can directly PR my repository, and will land here.

**Comment by xraver** [comment]
Ok.
I'll do tomorrow!
I will fork your repos and then merge with it.

Il Mar 27 Dic 2022, 19:13 Oliver Cervera ***@***.***> ha
scritto:

**Comment by balloob** [comment]
Please make sure to run the validate and test commands locally. That way it’s faster to find issues.

Our recommendation for new languages is to only translate the errors in `_common.yaml` and the files starting with `homeassistant_`. Big PRs can get stuck on missing tests for a file that could also be added later.

File: tests/it/climate_HassClimateGetTemperature.yaml [hunk 1]
```diff
@@ -1,6 +1,15 @@
 language: it
 tests:
-- sentences: []
-  intent:
-    name: HassClimateGetTemperature
-    slots: {}
+ - sentences:
+      - "Qual'è la temperatura?"
+      - "Che temperatura c'è"
```
Comment (alessandroias) [inline tests/it/climate_HassClimateGetTemperature.yaml:5]:
```suggestion
      - "Che temperatura c'è?"
```

**Review by alessandroias — CHANGES_REQUESTED** [review]
Thank you very much for carrying on this translation.
Very willing to help. Here a couple of oversights I found with a quick read.

**Comment by xraver** [comment]
I put my work in @auanasgheps patch.
@alessandroias if you want to contribuite I suggest you to wait the merge between @auanasgheps and mine and then you can revise it.
Is it fine for you?
@balloob yes thanks. Starting from the great work of @auanasgheps before sending the merge request I've tested all sentences as described in the manual usin pytest. This is very helpful!

best regards.

**Comment by auanasgheps** [comment]
Here we go, I merged the great work from @xraver who added a lot of intents and fixed some typos.

**Comment by auanasgheps** [comment]
There are a couple of stupid conflicts that I can't resolve, but the rest is looking good.

**Comment by xraver** [comment]
Thanks!
now we have all CI test passed!

**Comment by xraver** [comment]
> There are a couple of stupid conflicts that I can't resolve, but the rest is looking good.

where do you find them? i can try to fix

**Comment by xraver** [comment]
CLI notifies issue

Language: it
 - sentences/it/climate_HassClimateGetTemperature.yaml: invalid format: Sentences should only contain words and matching syntax. They should not contain punctuation. @ data['intents']['HassClimateGetTemperature']['data'][0]['sentences'][1]. Got "[(<what_is> | che)] <temp> [c'è]"
Sentences should only contain words and matching syntax. They should not contain punctuation. @ data['intents']['HassClimateGetTemperature']['data'][0]['sentences'][3]. Got "quanto (caldo | freddo) [(è | c'è | fa)] [(<in>|<the>) <area>]"
Sentences should only contain words and matching syntax. They should not contain punctuation. @ data['intents']['HassClimateGetTemperature']['data'][0]['sentences'][4]. Got "quanto [(è | c'è | fa)] (caldo | freddo) [(<in>|<the>) <area>]"
Sentences should only contain words and matching syntax. They should not contain punctuation. @ data['intents']['HassClimateGetTemperature']['data'][0]['sentences'][5]. Got "[(<what_is> | che)] <temp> [c'è] [<in>] <area>"

I think it refers to "c'è" but it's a standard sentence in italy...

**Comment by synesthesiam** [comment]
The punctuation validation is a bit too aggressive. I'm seeing what I can do :)

**Comment by xraver** [comment]
> The punctuation validation is a bit too aggressive. I'm seeing what I can do :)

No problem!

I've executed all test before commit and I have no issue.
I've repeated again and no issue...
/script/lint
/script/test
don't know why...

**Comment by synesthesiam** [comment]
Please rebase on `main` and try again

**Comment by xraver** [comment]
Ok all test passed!

**Comment by xraver** [comment]
Thanks!!!

**Pull request merged**